### ISSUE TITLE
Xcode 14.3 beta 2 fails to build SwiftUIBackports - iOS 14 API used

### ISSUE THREAD

**Issue by rhysm94** [shaps80/SwiftUIBackports#42]
*Have you read the [Contributing Guidelines](https://github.com/shaps80/.github/blob/master/CONTRIBUTING.md)?*

## General Information

 - Project version:
1.15.0
 
 - Platform/OS version: 
iOS 15+ (not entirely relevant to this)
 
 - IDE version:
Xcode 14.3 beta 2
  
 - Any related GitHub issues:

## Describe the bug
When attempting to build a project which imports SwiftUIBackports in Xcode 14.3 beta 2, the compiler fails on the file Toolbar.swift with the error `Namespace is only available in iOS 14.0 or newer`.
The same error doesn't happen in Xcode 14.2, but I'm not sure why that is. Accessing the same Toolbar.swift file and Cmd+right clicking the `Namespace` property wrapper shows that it *was* introduced in iOS 14, and the file shouldn't have been successfully building while the package targets iOS 13+.

## Steps to reproduce
Add SwiftUIBackports as a dependency to a project targeting any version of iOS with SwiftUI, and attempt to build in Xcode 14.3 beta 2.

## Expected behavior
SwiftUIBackports to build correctly

## Screenshots
*If applicable, add screenshots, gifs, or videos to help explain your problem.*
<img width="268" alt="image" src="https://user-images.githubusercontent.com/11096937/222452573-418df12e-6837-40f2-89dc-69b0b1ad375a.png">
<img width="952" alt="image" src="https://user-images.githubusercontent.com/11096937/222452622-09cead49-0d0b-49dd-86ff-badc00dc0327.png">

## Additional context
In the generated header for Namespace, it's attributed like this:
```swift
@available(iOS 14.0, macOS 11.0, tvOS 14.0, watchOS 7.0, *)
@frozen @propertyWrapper public struct Namespace : DynamicProperty {
```
So I'm really confused as to how it built/worked before 😅

**Comment by shaps80** [comment]
Great point! That shouldn't be there actually! Wow, no idea what's happened here. Thank you!

**Comment by rhysm94** [comment]
Closed with #44

**Comment by shaps80** [comment]
Thanks for fixing @rhysm94 – Now available in `v1.15.1`

**Comment by rhysm94** [comment]
No problem @shaps80, glad to help!
